content/messaging/contents.lr

```diff
@@ -0,0 +1,133 @@
+_model: topic
+---
+title: Instant Messaging
+---
+description: Instant Messaging and Tor
+---
+seo_slug: tor instant messaging
+---
+key: 14
+---
+body:
+
+Several instant messaging protocols and pieces of IM software are compatible with Tor. This page aims to document the most widely used and commonly accepted forms of anonymous instant messaging, as well as security advisories of potential vulnerabilities.
+
+#### Reasonably secure software
+##### Internet Relay Chat (IRC) clients
+
+For general information on IRC security, see [IRC](https://trac.torproject.org/projects/tor/wiki/doc/TorifyHOWTO/IRC). Guides on specific IRC clients:
+* [HexChat](https://trac.torproject.org/projects/tor/wiki/doc/TorifyHOWTO/HexChat)
+* [irssi](https://trac.torproject.org/projects/tor/wiki/doc/TorifyHOWTO/irssi)
+* [WeeChat](https://trac.torproject.org/projects/tor/wiki/doc/TorifyHOWTO/WeeChat)
+
+##### Other
+###### Signal Messenger
+
+**Warning:** While Signal does have a desktop version and provides strong encryption by default as well as location anonymity when paired with Tor, maintained access to a valid phone number is required to use it. Signal only requires one verification message in order to work, but if access to the physical number is lost, someone else could hypothetically take over the account. This can be done more securely by using a paid online SMS service, purchasing a pay-as-you-go phone with cash from a local store or another pseudo-anonymous choice. **DO NOT USE FREE SERVICES / TEMPORARY NUMBERS!**
```

Comment:
This strings will become too long to translate. Please add  linebreaks between sentences:
**Warning:**
While Signal does have a desktop version and provides strong encryption by default as well as location anonymity when paired with Tor, maintained access to a valid phone number is required to use it.
Signal only requires one verification message in order to work, but if access to the physical number is lost, someone else could hypothetically take over the account.
This can be done more securely by using a paid online SMS service, purchasing a pay-as-you-go phone with cash from a local store or another pseudo-anonymous choice.
**DO NOT USE FREE SERVICES / TEMPORARY NUMBERS!**

This way the content will still appear in one line, but the sentences will be separated and easier to reuse and translate. This in the end means better translations!

Reply:
ok, let's make it easier for translators.